pyproject.toml

```diff
@@ -6,7 +6,6 @@ authors = ["OpenHands"]
 license = "MIT"
 readme = "README.md"
 repository = "https://github.com/All-Hands-AI/OpenHands"
-include = ["poetry.lock"]
```

Comment:
any idea why we want to remove this? - this will be helpfulto pin-down packages inside runtime though

Reply:
It cannot be included at root level. It's still used, but in my `build.sh`, it's copied to `openhands` directory.

Why not root level? It will pollute `site-packages` folder and might have conflicts with other libraries.

![image](https://github.com/user-attachments/assets/e8820076-8d03-4922-a2ee-4eaa69049bdc)

Reply:
If you look at https://files.pythonhosted.org/packages/76/7c/f142053766542a3bf09a78de4e485246684d5da7a7fffc76f53eb1176565/openhands_ai_test-0.9.6.tar.gz,

you can see `openhands` directory now contains "poetry.lock" and "pyproject.toml".